docs/user-guide/configuration/sourcePaths.md

```diff
@@ -48,6 +48,10 @@ jobs:
 
 In this example, the job `main` will be triggered if there are any changes to files under `src/app/` or the `screwdriver.yaml` file (like on `src/app/main.js`, `src/app/package.json`, etc.). The `main` job will **not**, however, be triggered on changes to `README.md`, `test/`, or `src/other/`.
 
+### Matched source path
+
+Screwdriver will expose the matched souce path in an environment variable `SD_SOURCE_PATH`. This can be used to write scripts which depends on the source path that triggered the build.
```

Comment:
source* path

Reply:
`the source path(s) that triggered this build to run`?

not sure what makes more sense